website_sale_template_adj/controllers/__init__.py

```diff
@@ -0,0 +1 @@
+from . import main
```

Comment:
Delete this file and the controllers folder.

Reply:
I got it!